cmd/vclusterctl/cmd/delete.go

```diff
@@ -31,10 +33,11 @@ type DeleteCmd struct {
 	DeleteNamespace     bool
 	AutoDeleteNamespace bool
 
-	rawConfig  *clientcmdapi.Config
-	restConfig *rest.Config
-	kubeClient *kubernetes.Clientset
-	log        log.Logger
+	IgnoreNotFound bool
```

Comment:
Why uppercase for IgnoreNotFound? Is this variable being exported i.e. being used outside the scope of this package?

Reply:
My bad, did not realize this. Thanks for pointing it out!

Reply:
Hadn't realised this. Thanks for pointing it out!